portfolio/src/main/webapp/index.html

```diff
@@ -134,7 +136,15 @@ <h2>Misc</h2>
     </div>
     <!--Comments-->
     <div class="comment_background">
-      <div id="comments_sec"></div>
+      <div id="comments_sec">
+        <h2>Leave a Comment!</h2>
+        <form action="/comments" method="POST">
+          <textarea name="comments-input"class="form-control" rows="6";></textarea>
+          <br><input type="submit">
```

Comment:
nit: `input` can be on its own line after the line-break. You could probably also omit the `<br>` and handle the layout with CSS (probably using the `display` property).